bai-tap-buoi-11/main.js

```diff
@@ -0,0 +1,127 @@
+// Bài 1:
+// Cách 1:
+const getRepeatArray = (array, number) => {
+  for (let i = 0; i < number; i++) {
+    console.log(array)
+  }
+}
+
+// Cách 2:
+const getrepeatArrayVer2 = (array, number) => {
+  let index = 0
+  while (index < number) {
+    console.log(array)
+    index++
+  }
+}
+
+// Bài 2:
+const getReverseArray = array => {
+  let reverseArray = []
```

Comment:
```suggestion
  const reverseArray = []
```
`reverseArray` ko bị gán lại, do vậy phải khai báo là `const`